src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -207,7 +208,43 @@ const NotesSection = ({ selectedRelease }) => {
 	)
 }
 
+/**
+ * @TODO replace with InlineAlert
```

Comment:
❓ Is `InlineAlert` a new HDS component?

Reply:
It's not new, we just haven't built it yet. 😊

Reply:
We could link to the HDS docs in the comment to have a reference for what this should be?